packages/i18n/locales/en.json

```
@@ -159,6 +159,9 @@
       "previous": "Previous"
     },
     "common": {
+      "banner": {
+        "hide": "Hide this banner"
```

This feels more appropriate to me. Dismiss is used by [bootstrap](https://getbootstrap.com/docs/5.3/components/alerts/) and [MUI](https://mui.com/material-ui/react-snackbar/)

```suggestion
        "hide": "Dismiss this banner"
```